docs/troubleshoot.md

```diff
@@ -6,11 +6,36 @@ indent: true
 ---
 # Troubleshooting
 
+* [Using the trace command](#using-the-trace-command)
 * [Resource Status and Conditions](#resource-status-and-conditions)
 * [Crossplane Logs](#crossplane-logs)
 * [Pausing Crossplane](#pausing-crossplane)
 * [Deleting a Resource Hangs](#deleting-a-resource-hangs)
 
+## Using the trace command
+
+[Crossplane CLI](https://github.com/crossplaneio/crossplane-cli) trace
+command provides a holistic view for a particular object and related
+ones to ease debugging and troubleshooting process. It finds the
+relevant Crossplane resources for a given one and provides detailed
+information as well as an overview indicating what could be wrong.
+
+Usage:
+```
+$ kubectl crossplane trace TYPE[.GROUP] NAME [-n| --namespace NAMESPACE]
```

Comment:
We have been moving away from putting the `$ ` shell prompt at the beginning of sample commands, so that the commands are easier to copy and paste. For **this** one it's probably fine since it's usage, but for the examples, I would recommend removing the `$ `.